lib/kafo/hooking.rb

```diff
@@ -44,15 +45,15 @@ def loaded?
       @loaded
     end
 
-    def execute(group)
+    def execute(group, log_stage: true)
       logger = Logger.new(group)
-      logger.info "Executing hooks in group #{group}"
+      logger.info "Executing hooks in group #{group}" if log_stage
       self.hooks[group].keys.sort_by(&:to_s).each do |name|
         hook = self.hooks[group][name]
         result = HookContext.execute(self.kafo, logger, &hook)
         logger.debug "Hook #{name} returned #{result.inspect}"
       end
-      logger.info "All hooks in group #{group} finished"
+      logger.info "All hooks in group #{group} finished" if log_stage
```

Comment:
With #278 now merged, this can now be updated to use the level `notice`. The same applies a few lines above.